### DISCUSSION TITLE
docs at the website

### DISCUSSION THREAD

**Discussion by MeztliRA** [benawad/dogehouse#2143] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
there should be documentation at the website

**Describe the solution you'd like**
add documentation maybe at dogehouse.tv/docs

when theres a public api theres should be documentation for it to at the website

**Describe alternatives you've considered**
none

**Additional context**

**Comment by dk-raw** [comment]
Documentation for what? Contributing? If that's the case, there is no need for documentation on how to contribute on the main website.

**Comment by theotterlord** [comment]
I believe they mean this:

> when theres a public api theres should be documentation for it to at the website

so possibly for the endpoints/kebab

**Comment by MeztliRA** [comment]
> Documentation for what? Contributing? If that's the case, there is no need for documentation on how to contribute on the main website.

for the site, and also when theres a public api maybe also for that to

**Comment by dk-raw** [comment]
I personally think that GitHub is more suited for API docs

**Comment by theotterlord** [comment]
I think the only reason for a documentation site would be for usage docs. the rest can exist on github

**Comment by dk-raw** [comment]
I don't think we need usage docs. DogeHouse is very simple to use and understand.

**Comment by vi241** [comment]
it is

**Comment by vi241** [comment]
but a guide can be helpful if someone has a problem

**Comment by dk-raw** [comment]
We could have something like popups over each button to explain how it works, but those are very very annoying.

**Comment by vi241** [comment]
i think more of a help section in the settings

**Comment by vi241** [comment]
or a page like the github usage docs

**Comment by vi241** [comment]
but we can display the popups when the user hovers over the button

**Comment by dk-raw** [comment]
> but we can display the popups when the user hovers over the button

No popups, they can be quite annoying. A help section would be muck more elegant and suited.

**Comment by dk-raw** [comment]
I will open a discussion for this => https://github.com/benawad/dogehouse/discussions/2120

**Comment by vi241** [comment]
> > but we can display the popups when the user hovers over the button
> 
> No popups, they can be quite annoying. A help section would be muck more elegant and suited.

i agree

**Comment by samarmohan** [comment]
But no one is going to go to dogehouse.tv just to see some public api docs. We can have a pop up that tells people they can contribute.

**Comment by BronzW** [comment]
It's not a developers platform

**Comment by dk-raw** [comment]
> But no one is going to go to dogehouse.tv just to see some public api docs. We can have a pop up that tells people they can contribute.

No pop ups, something like a discrete button where the log out button is located.

**Comment by samarmohan** [comment]
I meant an alert or something, but that seems nice. The current GitHub button is tiny.

maybe when you click your profile the GH link shows under the discord one.

**Comment by dk-raw** [comment]
It's fine as it is, only as tiny percentage of users are actually willing to and know how to contribute.

**Comment by OkiStuff** [comment]
yes please we need it. im trying to write an API wrapper for dogehouse but its confusing to try to figure out how to use the api

**Reply by samarmohan** [reply]
Using which language? If it's JS/TS don't. There are already a lot such as https://github.com/fuwwy/moonstone, kebab (the official one), and https://github.com/dogegarden/dogehouse.js

**Comment by amitojsingh366** [comment, marked as answer]
https://doc.dogehouse.tv
